be/src/olap/tablet_schema_cache.h

```diff
@@ -49,6 +50,9 @@ class TabletSchemaCache {
             pb.ParseFromString(key);
             tablet_schema_ptr->init_from_pb(pb);
             _cache[key] = tablet_schema_ptr;
+            DorisMetrics::instance()->tablet_schema_cache_count->increment(1);
+            DorisMetrics::instance()->tablet_schema_cache_memory_bytes->increment(
+                    tablet_schema_ptr->mem_size());
```

Comment:
mem_size method is not accurate. And I find it is very tick.
   std::vector<TabletColumn> _cols;
    std::vector<TabletIndex> _indexes;
    std::unordered_map<std::string, int32_t> _field_name_to_index;
    std::unordered_map<int32_t, int32_t> _field_id_to_index;

TableSchema has a lot of properties. 

I think you could add a property in TabletSchema and using scoped_mem_tracker to set its size during init_from_pb.

And you could remove all mem_size method. I find it is useless before.

Reply:
good suggestion, fixed